packages/table/src/Row.tsx

```diff
@@ -50,8 +50,8 @@ const modeStyles = {
       background-color: ${palette.gray.light2};
       color: ${palette.gray.base};
       cursor: not-allowed;
-      box-shadow: 0 -1px 0 inset ${palette.gray.light1},
-        0 1px 0 inset ${palette.gray.light1};
+      border-top: 1px inset ${palette.gray.light1};
+      border-bottom: 1px inset ${palette.gray.light1};
```

Comment:
Not exactly the same, but effectively the same. Using border so I can have the cell's border over this border.